validator/flags/flags.go

```diff
@@ -159,21 +159,36 @@ var (
 		Usage: "Display raw eth1 tx deposit data for validator accounts-v2",
 		Value: false,
 	}
-	// AccountsFlag for non-interactive usage of accounts exporting, sets a list of account names or all to be exported.
-	AccountsFlag = &cli.StringSliceFlag{
-		Name:  "accounts",
-		Usage: "List of account names to export, or \"all\" to backup all accounts",
-	}
 	// NumAccountsFlag defines the amount of accounts to generate for derived wallets.
 	NumAccountsFlag = &cli.Int64Flag{
 		Name:  "num-accounts",
 		Usage: "Number of accounts to generate for derived wallets",
 		Value: 1,
 	}
+	// DeletePublicKeysFlag defines a comma-separated list of hex string public keys
+	// for accounts which a user desires to delete from their wallet.
+	DeletePublicKeysFlag = &cli.StringFlag{
+		Name:  "delete-public-keys",
+		Usage: "Comma-separated list of public key hex strings to specify which validator accounts to delete",
+		Value: "",
+	}
+	// BackupPublicKeysFlag defines a comma-separated list of hex string public keys
+	// for accounts which a user desires to backup from their wallet.
+	BackupPublicKeysFlag = &cli.StringFlag{
+		Name:  "backup-public-keys",
+		Usage: "Comma-separated list of public key hex strings to specify which validator accounts to backup",
+		Value: "",
+	}
+	// BackupsPasswordFile for encrypting accounts a user wishes to back up.
+	BackupsPasswordFile = &cli.StringFlag{
+		Name:  "backups-password-file",
```

Comment:
A bit strange that `Backup**S**PasswordFile` has been used, instead of `BackupPasswordFile` (which would have been consistent with other backup related flags/options. It seems `--backup-password-file` would have been more natural here.